releasenotes/notes/add-classical-predecessors-9ecef0561822e934.yaml

```diff
@@ -0,0 +1,4 @@
+---
+features:
+  - |
+  Add some info about classical_predecessor method
```

Comment:
Could you update this release note with a more detailed explanation about the 2 new methods that have been added? It would be great to also add a code example

Reply:
I did it